### ISSUE TITLE
(3.2.10) would be convenient to have gitRepoRoot displayed in "select branch" window

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#942]
When I looked at this list of branches I noticed that there was a branch name "missing" and I was looking at "my git repo on disk" where clearly the "missing branch" was present ... the problem was that the GsDevKit_upgrade had been loaded from another disk location and having the disk location in my face would have helped me notice my mistake earlier:
![image](https://user-images.githubusercontent.com/623951/187987852-e0ae552d-ebbe-45a1-bdb7-e06066ecfc91.png)

Alternatively, perhaps a status (popup) view that listed all of the information about the project would be useful ... I know I can open a project browser or navigate to the project in an existing project browser or select the project tab in an existing project browser, but the project browser is such a large window to open on demand and I tend to use existing project browser to manage my work state and unless I'm lucky enough to have the project browser already pointing at the project (the tab is light weight option) I have to choose to open a browser or open a new browser tab and navigate or lose state and navigate ...

**Comment by LisaAlmarode** [comment]
I agree that more information would be useful here, but not a priority for 2022.

**Comment by dalehenrich** [comment]
I tend to feel that this path is important especially when trying to help someone who is having problems ... so @LisaAlmarode please reconsider your disposition, with this in mind

**Comment by ericwinger** [comment]
@dalehenrich In the browser, one can look at the Project tab to get the projectUrl. Does that get you what you need? 
![image](https://user-images.githubusercontent.com/36212516/197633535-dc4e7f0f-2b0c-4c52-9a24-8bf7aaf7b016.png)

**Comment by dalehenrich** [comment]
It does ... the second part of the issue is that it is a very heavyweight operation to open a ginormous project browser to see that information ... but if a more lightweight solution is not a priority ... I can patiently wait for a better solution ...

**Comment by ericwinger** [comment]
Fixed in 3.2.13. I've added the project url to the checkout branch and tag dialogs. Need Lisa's input on aesthetics.

**Comment by LisaAlmarode** [comment]
It looks fine to me, but this wasn't something I felt very strongly about. 

Could consider putting this information below the list pane, since it breaks the "flow" from the window title to the list, but if you've just selected the menu item checkout branch and you can't figure out the contents of the list pane, rearranging the field is unlikely to solve the problem.
